scripts/_ecs.sh

```diff
@@ -47,6 +49,31 @@ function _ecs_run() {
     aws ecs run-task --cli-input-json "file://terraform/20-app/ecs-jobs/${job}.json" | jq ".tasks[0].taskArn"
 }
 
+function _ecs_run_and_wait() {
+    local job=$1
+
+    if [[ -z ${job} ]]; then
+        echo "Job is required" >&2
+        return 1
+    fi
+
+    echo Starting job....
+
+    current_cluster_name=$(_get_current_cluster)
+
+    task_arn=$(aws ecs run-task --cli-input-json "file://terraform/20-app/ecs-jobs/${job}.json" | jq -r ".tasks[0].taskArn")
+    echo "Job (${task_arn}) is now running, waiting for completion..."
+
+    aws ecs wait tasks-stopped --tasks ${task_arn} --cluster ${current_cluster_name}
```

Comment:
We need the cluster as well as the task arn otherwise, the underlying command will use the default cluster